vandflaske.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+	
+	
+<title>Untitled Document</title>
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script src="test.js"></script>
+	<link href="https://fonts.googleapis.com/css?family=Montserrat|Open+Sans|Roboto&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="vandflaskecss.css">
+</head>
+<body>
+  
+
+	
+	<img src="logo3.png"  alt="dropit" class="logo">
+	
+<article class="container">
```

Comment:
<article class="container">
  <nav class="desk">
    <ul>
      <li class="active"><a href="#">Hjem</a></li>
      <li><a href="#">Shop</a></li>
      <li><a href="#">Forhandlere</a></li>
      <li><a href="#">FAQ</a></li>
    </ul>
  </nav>

  <nav class="mob">
    <div class="hamburger-btn">
        <i class="fa fa-bars" aria-hidden="true"></i>
        <i class="fa fa-times" aria-hidden="true"></i>
    </div>

    <ul>
      <li class="active"><a href="#">Hjem</a></li>
      <li><a href="#">Shop</a></li>
      <li><a href="#">Forhandlere</a></li>
      <li><a href="#">FAQ</a></li>
    </ul>
  </nav>
</article> 

change this